rules/expiring-todo-comments.js

```diff
@@ -53,7 +53,7 @@ function parseTodoWithArguments(string, {terms}) {
 	return rawArguments
 		.split(',')
 		.map(argument => parseArgument(argument.trim()))
-		.reduce((groups, argument) => {
+		.reduce((groups, argument) => { // eslint-disable-line unicorn/no-reduce
```

Comment:
We can't just disable this rule, we need rewrite it. If we can't make rules work in our own codebase, how can we ask people to use it? That's also why we have this `npm run lint` test.

But we can do this later, before we merge.